.gitignore

```diff
@@ -164,14 +164,15 @@ cython_debug/
 # vscode related
 .vscode
 
-# Shark related artefacts
+# Shark related artifacts
```

Comment:
Nit: "artefact" is the British spelling, though I think we're using American spellings.

Reply:
Yeah. Not really necessary but makes me feel better. :)